app/controllers/api/background_services_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Api::BackgroundServicesController < ActionController::Base
+  before_action :check_api_key
+
+  # Fetch last 6 month refund records and sync with
+  # payment_braintree_transactions table
+  def sync_braintree_refunds
```

Comment:
Let's put this in `app/controllers/api/payment/braintree_controller.rb` as a `def refund` since that's where we're placing `Braintree` related webhooks as well.

Reply:
ok